chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -689,3 +689,11 @@ realWorldTesting.report.missingPlansWarning:WARNING: Listings eligible for Real
 realWorldTesting.report.missingPlansError:ERROR: Listings eligible for Real World Testing in %s must have a Plans URL by %s.
 realWorldTesting.report.missingResultsWarning:WARNING: Listings eligible for Real World Testing in %s must have a Results URL by %s.
 realWorldTesting.report.missingResultsError:ERROR: Listings eligible for Real World Testing in %s must have a Results URL by %s.
+
+svap.delete.listingsExist=This SVAP is attested to by %s Listings: %s
+svap.edit.deletedCriteria.listingsExist=This SVAP and Criteria %s is attested to by %s Listings: %s
+svap.edit.noCriteria=An SVAP requires at least one Criterion associated with it.
+svap.edit.emptyApprovedStandardVersion=Approved Standard Version is required.
+svap.edit.emtptyRegulatoryTextCitation=Regulatory Text Citation is required.
+svap.edit.duplicate=Regulatory Text Citation and Approved Standard Version matches an existing SVAP.
+ 
```

Comment:
We have a tool that's finding these things. Are we going to address them?

Reply:
Sorry...  I messed up.

Reply:
Not worried about messing up, was just wondering if we had been planning on addressing the auto-generated suggestions or not. I'm not going to hold things up based on those auto-generated ones, unless we decide we want to